docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -100,13 +139,14 @@ To run this guide, make sure to be in **Implement** mode to specify the technica
 
 1. A **start event** is automatically added to the canvas. Click it to display configuration and append options.
 2. Click the rectangular **Append Task** icon to append a task.
```

Comment:
<img width="806" alt="image" src="https://github.com/user-attachments/assets/f0d29da9-95ba-4212-9c21-dfc79c921919">
These instructions should be updated to account for the new context pad. The user has to hover over the blue circle to the right, then click the rectangular append task icon. This applies for all instructions- they should be double-checked